views/main.html

```diff
@@ -327,7 +327,7 @@
   <div class="modal fade" tabindex="-1" role="dialog" id="tagModal">
     <div style="position:fixed;left:0;top:0">
       <span ng-repeat="shot in filmDB[fetchedTagsForFilm].sshot">
-        <img src="{{shot}}" class="img-responsive" style="height:20vh;width:auto;">
+        <img ng-src="{{shot}}" class="img-responsive" style="height:20vh;width:auto;">
```

Comment:
When the page initially loads (before angular) it complains that {{shot}} doesn't exist. This prevents that.